### ISSUE TITLE
About the robust acc and clean acc inconsistency!!

### ISSUE THREAD

**Issue by motoight** [deepmind/deepmind-research#313]
Hi, there. I am trying to test how your pretrain robust model under the attack of AutoAttack.
And I downloaded your pretrain model [here](https://github.com/deepmind/deepmind-research/tree/master/adversarial_robustness#downloading-a-model). And I got got the result below, I am confused that this pretrain model result is different from the result you list [here](https://github.com/deepmind/deepmind-research/tree/master/adversarial_robustness#downloading-a-model).

<html>
<body>

dataset | norm | radius | architecture | extra data | clean | robust | link
-- | -- | -- | -- | -- | -- | -- | --
CIFAR-10 | ℓ∞ | 8 / 255 | WRN-70-16 | ✓ | 91.10% | 65.88% | jax, pt
CIFAR-10 | ℓ∞ | 8 / 255 | WRN-28-10 | ✓ | 89.48% | 62.80% | jax, pt

</body>
</html>

My test code is paste below:

```
import torch
transform_list = [transforms.ToTensor()]
transform_chain = transforms.Compose(transform_list)
item = datasets.CIFAR10(root='./database/CIFAR10/', train=False, transform=transform_chain, download=True)
test_loader = torch.utils.data.DataLoader(item, batch_size=1000, shuffle=False, num_workers=0)
l = [x for (x, y) in test_loader]
x = torch.cat(l, 0)
l = [y for (x, y) in test_loader]
y = torch.cat(l, 0)
ckpt = torch.load('./pretrain_model/Gowal2020/cifar10_linf_wrn70-16_with.pt')
from model_zoo import WideResNet
r_model = WideResNet(num_classes=10, depth=70, width=16)
r_model.load_state_dict(ckpt)
r_model.cuda()
r_model.eval()
from autoattack import AutoAttack

model_name = 'cifar10_linf_wrn70-16_with.pt'
adversary = AutoAttack(r_model, norm='Linf', eps=8./255., log_path=f'{model_name}_log_file.txt',
                           version='custom')
adversary.attacks_to_run = ['apgd-ce']

with torch.no_grad():
      adv_x, adv_y = adversary.run_standard_evaluation(x[:100], y[:100], bs=100, return_labels=True)
```

My result is:

> Files already downloaded and verified 
> using custom version including apgd-ce 
> initial accuracy: 73.00% 
> apgd-ce - 1/1 - 30 out of 73 successfully perturbed  
> robust accuracy after APGD-CE: 43.00% (total time 247.8 s)  
> max Linf perturbation: 0.03137, nan in tensor: 0, max: 1.00000, min: 0.00000  
> robust accuracy: 43.00%  

I've noticed that you have down normalization in your wideresent model in [model_zoo.py](https://github.com/deepmind/deepmind-research/blob/master/adversarial_robustness/pytorch/model_zoo.py#L154)

I am wondering if I didn't notice or miss any operation to get a fine results. Tnank you for sharing great and powerful pretrain model!

**Comment by sgowal** [comment]
Hi motoight,

Please check the exact loading procedure in [pytorch/eval.py](https://github.com/deepmind/deepmind-research/blob/master/adversarial_robustness/pytorch/eval.py). In particular, you may have forgotten to specify the activation to be swish.

Let us know if you run into more trouble.

Sven

**Comment by motoight** [comment]
Thanks！
I switch activation fn form RELU to Swish and got the results below(WRN-28-10)!
Thanks again for sharing your excellent jobs!

> Files already downloaded and verified
> setting parameters for standard version
> using standard version including apgd-ce, apgd-t, fab-t, square
> initial accuracy: 89.90%
> apgd-ce - 1/5 - 60 out of 200 successfully perturbed
> apgd-ce - 2/5 - 69 out of 200 successfully perturbed
> apgd-ce - 3/5 - 48 out of 200 successfully perturbed
> apgd-ce - 4/5 - 52 out of 200 successfully perturbed
> apgd-ce - 5/5 - 18 out of 99 successfully perturbed
> robust accuracy after APGD-CE: 65.20% (total time 367.7 s)
> apgd-t - 1/4 - 7 out of 200 successfully perturbed
> apgd-t - 2/4 - 8 out of 200 successfully perturbed
> apgd-t - 3/4 - 7 out of 200 successfully perturbed
> apgd-t - 4/4 - 1 out of 52 successfully perturbed
> robust accuracy after APGD-T: 62.90% (total time 2699.8 s)
> fab-t - 1/4 - 0 out of 200 successfully perturbed
> fab-t - 2/4 - 0 out of 200 successfully perturbed
> fab-t - 3/4 - 0 out of 200 successfully perturbed
> fab-t - 4/4 - 0 out of 29 successfully perturbed
> robust accuracy after FAB-T: 62.90% (total time 5853.9 s)
> square - 1/4 - 0 out of 200 successfully perturbed
> square - 2/4 - 0 out of 200 successfully perturbed
> square - 3/4 - 0 out of 200 successfully perturbed
> square - 4/4 - 0 out of 29 successfully perturbed
> robust accuracy after SQUARE: 62.90% (total time 9861.3 s)
> max Linf perturbation: 0.03137, nan in tensor: 0, max: 1.00000, min: 0.00000
> robust accuracy: 62.90%
